gvsbuild/projects/webrtc_audio_processing.py

```diff
@@ -0,0 +1,24 @@
+from gvsbuild.utils.base_builders import Meson
+from gvsbuild.utils.base_expanders import Tarball
+from gvsbuild.utils.base_project import Project, project_add
+
+
+@project_add
+class WebrtcAudioProcessing(Tarball, Meson):
+    def __init__(self):
+        Project.__init__(
+            self,
+            "webrtc-audio-processing",
+            repository="https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/",
+            version="1.3",
+            archive_url="https://freedesktop.org/software/pulseaudio/webrtc-audio-processing/webrtc-audio-processing-{version}.tar.gz",
+            hash="95552fc17faa0202133707bbb3727e8c2cf64d4266fe31bfdb2298d769c1db75",
+            dependencies=["meson", "ninja"],
+        )
+
+    def build(self):
+        Meson.build(self)
+        self.install(r".\COPYING share\doc\webrtc-audio-processing")
+        self.install(
+            r".\webrtc\LICENSE .\webrtc\PATENTS share\doc\webrtc-audio-processing\webrtc"
```

Comment:
I would guess that theses aren't be necessary.

Reply:
Hi @KyleSiefring, why do you think so? You may be right, I'm just trying to understand.

Reply:
Just not certain about the legal stuff.

[COPYING](https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/blob/master/COPYING?ref_type=heads) and [LICENSE](https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/blob/master/webrtc/LICENSE?ref_type=heads) are the same except for work author. This difference should be reason enough to include both, but I didn't notice it before.

[PATENTS](https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/blob/master/webrtc/PATENTS?ref_type=heads) doesn't require any attribution. Including both should be fine, even if not legally necessary.

Reply:
I think either way is fine... I think in the past we manually added installation for `COPYING` to ease creation of attribution documents when redistributing the binaries in an application, but looking at a few different modules it is a mixed bag.